project/BinaryIncompatibilities.scala

```diff
@@ -12,6 +12,8 @@ object BinaryIncompatibilities {
   )
 
   val JSEnvsTestKit = Seq(
+    ProblemFilters.exclude[MissingClassProblem]("org.scalajs.jsenv.test.NodeJSWithCustomInitFilesTest"),
+    ProblemFilters.exclude[MissingClassProblem]("org.scalajs.jsenv.test.PhantomJSWithCustomInitFilesTest")
```

Comment:
There should be a comment saying "Breaking", as this is actually breaking.
Also, 4 spaces wrt `val`, and wrap with 4 more spaces after `(`.